src/main/java/de/doubleslash/keeptime/controller/Controller.java

```diff
@@ -184,17 +184,47 @@ public void editProject(final Project projectToBeUpdated, final Project newValue
       model.getProjectRepository().saveAll(changedProjects);
    }
 
+   public void editWork(final Work workToBeEdited, final Work newValuedWork) {
+      LOG.info("Changing work '{}' to '{}'.", workToBeEdited, newValuedWork);
+      int index = model.getPastWorkItems().size() - 1;
+      for (int i = 0; i < model.getPastWorkItems().size(); i++) {
```

Comment:
Resorting the pastWorkitems manualy is extra code we do not need. I think its just for the timeline to be displayed correctly?

Better use a sorted list (see e.g. https://github.com/doubleSlashde/KeepTime/blob/develop/src/main/java/de/doubleslash/keeptime/model/Model.java#L61 ) or sort if if we need it sorted.

Reply:
sorted list is impossible because it is an Observable list.

if we resort it everytime we need it sorted would sort every second to update timeline